db/python/layers/seqr.py

```diff
@@ -408,7 +408,7 @@ async def post_es_index_update(self, session: aiohttp.ClientSession, url: str, p
             headers=headers,
         )
         resp.raise_for_status()
-        return await resp.text()
+        return f'Updated ES index {post_json["elasticsearchIndex"]}: {resp.status} - {resp.reason}'
```

Comment:
This is fine, but IMO you could be more defensive (using `.get`) to stop the whole thing failing if the key doesn't exist.